.github/workflows/workflow.yml

```diff
@@ -29,12 +29,12 @@ jobs:
       - name: Pipenv install requirements and check it can be locked
         run: |
           pipenv --clear
-          pipenv install --dev --skip-lock
-          pipenv run pip install -e .[all]
-          pipenv run pip install --no-deps file:plugins/aea-ledger-ethereum
-          pipenv run pip install --no-deps file:plugins/aea-ledger-cosmos
-          pipenv run pip install --no-deps file:plugins/aea-ledger-fetchai
-          pipenv lock
+          time pipenv install --verbose --dev --skip-lock
+          time pipenv run pip install -e .[all]
+          time pipenv run pip install --no-deps file:plugins/aea-ledger-ethereum
+          time pipenv run pip install --no-deps file:plugins/aea-ledger-cosmos
+          time pipenv run pip install --no-deps file:plugins/aea-ledger-fetchai
+          time pipenv lock
```

Comment:
this will log the timing. Please have a look at the logs of [`common_checks_1` on CI here](https://github.com/valory-xyz/open-aea/actions/runs/3313301571/jobs/5471097240)

here a snippet for the lazy